Source/WebCore/rendering/RenderLayer.h

```diff
@@ -1283,9 +1283,9 @@ class RenderLayer final : public CanMakeSingleThreadWeakPtr<RenderLayer>, public
     bool m_hasNotIsolatedBlendingDescendantsStatusDirty : 1;
     bool m_repaintRectsValid : 1;
 
-    bool m_intrinsicallyComposited : 1;
-    bool m_hasIntrinsicallyCompositedDescendants : 1;
-    bool m_hasIntrinsicallyCompositedDescendantsStatusDirty : 1;
+    bool m_intrinsicallyComposited : 1 { false };
+    bool m_hasIntrinsicallyCompositedDescendants : 1 { false };
+    bool m_hasIntrinsicallyCompositedDescendantsStatusDirty : 1 { false };
```

Comment:
Should this start as `true`?

Reply:
Righto, the dirty bit could start as true.